### DISCUSSION TITLE
Github profile does not show contributors

### DISCUSSION THREAD

**Discussion by tranquocviet226** [community/community#36853] [Other Feature Feedback, Questions, & Ideas] [answered]
In October, I have a contributor on git almost every day, but today when I access it, I see very few contributors

![image](https://user-images.githubusercontent.com/57559641/197097312-c0245fed-2199-4de6-80eb-1610e1c658f7.png)

**Comment by radar-blip** [comment]
Do you have private contirbutions set to > include on profile?

![profiles](https://user-images.githubusercontent.com/39196767/197098580-a0ed14eb-5f1b-453a-8148-4d1a787bf528.png)

**Comment by tranquocviet226** [comment, marked as answer]
oh no. It's my mistake.
I forgot I deleted that repository

**Comment by ossanna16** [comment]
Thank you for asking a good question @tranquocviet226, I'm glad you found our community 🙂
